Facebook-DP-Downloader/Readme.md

```diff
@@ -0,0 +1,22 @@
+# <center> Facebook Profile Picture Downloader </center>
+<center> Download the profile picture of any public profile on facebook by just it's fbid. </center>
+<br>
+
+## Modules used:
+---------------
+* os
+* request
+
+## How it works:
+---------------
+* Just run the script by entering the range of fbids (valid fb-user-id) and get the profile-pictures stored in your local file.
```

Comment:
What is this `fbid`? I know the full form but in readme files, this doesn't look good. Replace it with full name: Facebook id for every instance in the readme and the code